README.md

```diff
@@ -8,17 +8,19 @@ This action supports two ways of logging in:
 
 ## OpenID Connect (OIDC)
 
-> Support for OpenID Connect is currently in development and may not be available in your Octopus version just yet.
+> Support for OpenID Connect is currently being rolled out to Octopus Cloud and may not be available in your Octopus version just yet.
 
 Using OpenID Connect (OIDC) is the recommended way to login to Octopus from GitHub Actions. It allows the granting of short-lived access tokens for a service account in Octopus that can be used during your GitHub Actions workflow run, without needing to provision or store an API key.
 
-See [About security hardening with OpenID Connect](https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/about-security-hardening-with-openid-connect) for more information.
+For more information about using OpenID Connect in GitHub Actions see [about security hardening with OpenID Connect](https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/about-security-hardening-with-openid-connect).
 
 To login using OIDC:
 
--   Create a service account in Octopus with the permissions required.
--   Configure an OIDC identity that matches the GitHub Actions subject claim for your repository and workflow. See the [GitHub documentation](https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/about-security-hardening-with-openid-connect#defining-trust-conditions-on-cloud-roles-using-oidc-claims) for examples of the subject claim.
+-   Create a [service account](https://oc.to/ServiceAccount) in Octopus with the permissions required. Note that OIDC is only support for service accounts, not user accounts.
+-   Configure an OIDC identity for the service account that matches the GitHub Actions subject claim for your repository and workflow. See the [Octopus OIDC documentation](https://oc.to/ServiceAccountOidcIdentities) for more information.
+    -   Examples of the subject claims can be found in the [GitHub documentation](https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/about-security-hardening-with-openid-connect#defining-trust-conditions-on-cloud-roles-using-oidc-claims).
```

Comment:
Link text could be more descriptive to be clearer when read out of context. https://dequeuniversity.com/tips/link-text

Potentially something like: `GitHub OIDC claims documentation`